### PR TITLE
Introduce Che Theia about menu instead of Theia one

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?

It provides Eclipse Che logo and sha1 of che theia and theia

image

### What issues does this PR fix or reference?
N/A

Change-Id: I0baa118b7273226ecc479a15b840568fcc35cfaa
Signed-off-by: Florent Benoit <fbenoit@redhat.com>

### CODE REVIEW

File: extensions/eclipse-che-theia-about/package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@eclipse-che/theia-about",
```
Comment (mmorhun) [inline extensions/eclipse-che-theia-about/package.json]: It is up to you, but to me `@eclipse-che/che-theia-about` looks more consistent.

Comment (benoitf) [reply extensions/eclipse-che-theia-about/package.json]: @mmorhun I tried to be consistent with existing extensions
https://github.com/eclipse/che-theia/blob/master/extensions/eclipse-che-theia-plugin-ext/package.json#L2

https://github.com/eclipse/che-theia/blob/master/extensions/eclipse-che-theia-dashboard/package.json#L2

it seems they're using the pattern `@eclipse-che/theia-something`

**Comment by benoitf** [comment]
ci-build

File: extensions/eclipse-che-theia-about/package.json (hunk 1 above)
Comment (mmorhun) [reply extensions/eclipse-che-theia-about/package.json]: I see. My point was that `@eclipse-che` is namespace and then we need to have a component which is Che Theia About, like you have in class and file names: `AboutCheTheiaDialog`, `about-che-theia-dialog.ts`

**Pull request merged**